lib/identity_config.rb

```diff
@@ -0,0 +1,80 @@
+class IdentityConfig
```

Comment:
our goal is to move this to a gem like `identity-hostdata` right?

Reply:
definitely. My thinking was to wait until `identity-idp` was finished and see how PKI does with only using `IdentityConfig` before we commit to the abstraction.

Reply:
😌 ok great! Glad we're on the same page